docker/compose/withPostgres/docker-compose.yml

```diff
@@ -16,6 +16,7 @@ services:
 
   n8n:
     image: n8nio/n8n
+    depends_on: postgres
```

Comment:
unfortunately, this doesn't work because this assumes that the postgres container is ready as soon as the container starts. 
because of the db initialization process, postgres container is actually only ready after the `init-data.sh` script has executed, and the process has restarted after that.

I've added a correct health-check based condition [here](https://github.com/n8n-io/n8n/pull/3369/commits/a4b160d85769a817175ff12619396da006711623).